cpp/log/etcd_consistent_store_test.cc

```diff
@@ -147,11 +147,11 @@ class EtcdConsistentStoreTest : public ::testing::Test {
   void AddSequenceMapping(int64_t seq, const string& hash) {
     EntryHandle<SequenceMapping> mapping;
     Status status(store_->GetSequenceMapping(&mapping));
-    CHECK_EQ(Status::OK, status);
+    CHECK_EQ(::util::OkStatus(), status);
     SequenceMapping::Mapping* m(mapping.MutableEntry()->add_mapping());
     m->set_sequence_number(seq);
     m->set_entry_hash(hash);
-    CHECK_EQ(Status::OK, store_->UpdateSequenceMapping(&mapping));
+    CHECK_EQ(::util::OkStatus(), store_->UpdateSequenceMapping(&mapping));
```

Comment:
Might be worth creating a `CHECK_OK` macro, given how often this is done. Shame we weren't already using one as this would have been a much smaller change.